src/reducers/index.js

```diff
@@ -68,6 +69,11 @@ const combinedReducer = combineReducers({
   fontLoaded: createReducer(false, UPDATE_FONT_LOADED, 'fontLoaded'),
   textLabels: createReducer(true, TOGGLE_TEXT_LABELS, 'textLabels'),
   theme: createReducer('dark', TOGGLE_THEME, 'theme'),
+  ignoreLargeWarning: createReducer(
+    {},
```

Comment:
Shouldn't this have a Boolean default value here as the first argument?
```suggestion
    false,
```

Reply:
This is fixed in the latest commit